<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome To Pencil Producer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="makepencil">
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>20</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(41, 235, 239);</string>
    </property>
    <property name="text">
     <string>Make Pencil</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pencil Producer</string>
    </property>
   </widget>
   <widget class="QLabel" name="pencils_inv">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pencils in Inventory: 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buywood">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(193, 125, 17);
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Buy Wood
$1000.00 / 100.00m</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buygraphite">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(136, 138, 133);
color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Buy Graphite
$1500.00 / 100.00m</string>
    </property>
   </widget>
   <widget class="QLabel" name="bankbalance">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bank Balance</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalbalance">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>$145.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="woodleft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Wood Left: 1000.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="graphiteleft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Graphite Left: 1000.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="pencil_price">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pencil Price: 1.00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pencil_price_inc">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>89</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(115, 210, 22);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pencil_price_dec">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 0, 0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="total_pencils">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pencils Produced (to date): 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="demand">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Public Demand: 1.00</string>
    </property>
   </widget>
   <widget class="QLabel" name="autopencil">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>AutoPencil2000: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="pencil_rate">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rate: 0 pencils / minute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buy_APM">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>300</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(237, 212, 0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Buy More
$150.00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadBtn">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>371</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="intelligence">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Intelligence</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalintelligence">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upgradeAPM">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(25, 102, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Upgrade APM
50 Intelligence</string>
    </property>
   </widget>
   <widget class="QPushButton" name="marketing">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 230, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Unlock Marketing</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="debug">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="debugBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(25, 160, 255);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
